django/contrib/gis/geos/geometry.py

```diff
@@ -174,12 +174,12 @@ def from_gml(cls, gml_string):
     def __eq__(self, other):
         """
         Equivalence testing, a Geometry may be compared with another Geometry
-        or a WKT representation.
+        or a WKT or EWKT representation.
```

Comment:
As for me, I don't think that compare geometries with strings is a good idea, but once we have such feature I don't see a problem to use EWKT instead of WKT for it.